tests/functional_tests/p2p.py

```diff
@@ -158,6 +159,48 @@ def test_p2p_reorg(self):
             loops -= 1
             assert loops >= 0
 
+        # create a transaction which both daemons have in their pool
```

Comment:
This file has changed quite a bit on the release branch, probably a good idea to rebase and then adjust any changes you want to keep.